pipelines/config/dependency-builds.yml

```diff
@@ -249,6 +249,7 @@ dependencies:
     source_type: node
     copy-stacks:
       - bionic
+      - jammy
```

Comment:
Do we need the stack called `jammy`? I believe the copy to `bionic` stack was introduced to support paketo at some point (who don't use this pipeline anymore).
Question applies to all places with jammy reference

Reply:
I was not aware that this change is no longer used. We can eliminate this (I'll delete the references to bionic too).